include/analyzedphenotypes.validators.inc

```diff
@@ -831,9 +858,7 @@ function analyzedphenotypes_save_tsv_data($dataset, $job_id) {
     exit(3);
   }
 
-  $file_contents = analyzedphenotypes_datafileprop('get_contents', array(
-    'data_file' => $data_file,
-  ));
+  $file_contents = file_get_contents($file);
```

Comment:
We still don't want to read the whole file in here. See https://stackoverflow.com/questions/4857182/best-way-to-determine-if-a-file-is-empty-php. I would use `filesize()` below and then read the file line-by-line where you loop through the rows.